rapid_response_xblock/static/html/rapid.html

```diff
@@ -1,17 +1,25 @@
-<div class="rapid-response-block" data-staff="{{ is_staff }}" data-open="{{ is_open }}">
+<div class="rapid-response-block" data-open="{{ is_open }}">
   <h3 class="chart-title">Live Response</h3>
   <div class="rapid-response-content">
   </div>
   <div class="rapid-response-results">
   </div>
 
   <script type="text/template" id="rapid-response-toggle-tmpl">
-    <% if (is_staff && selectedRuns.length === 1) { %>
-      <p>
+    <% if (selectedRuns.length === 1) { %>
+      <div class="buttons-row">
         <button class="btn btn-brand problem-status-toggle">
           <%= is_open ? "Close" : "Open" %> Problem Now
         </button>
-      </p>
+        <div class="timer">
+          <%= pollMinutes %>m : <%= pollSeconds %>s
+        </div>
+        <div class="timer-spinner fa fa-spinner fa-pulse fa-spin <%= is_open ? "" : "hidden" %>">
+        </div>
+        <div class="timer-spinner-text <%= is_open ? "" : "hidden" %>">
+          Problem is Open
```

Comment:
"Waiting for responses" is probably more helpful here

Reply:
@roberthouse54 What do you think?

Reply:
I think "Problem is Open" is ok. Let's just leave it for now.